strings/pad.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/** FillOption Object */
+export interface FillOption {
+  /** Char to fill in */
+  char: string;
+  /** Side to fill in */
+  side: Side;
+  /** If strict, output string can't be greater than strLen*/
+  strict?: boolean;
+  /** char/string used to specify the string has been truncated */
+  strictChar?: string;
+  /** Side of truncate */
+  strictSide?: Side;
+}
+
+export enum Side {
```

Comment:
Maybe just type Side = “left” | “right” suffices

Reply:
That was the only point i was doubting. But yes, changing it.